scripts/core/exp-command-buffer.yml

```diff
@@ -307,6 +312,14 @@ params:
     - type: "const size_t*"
       name: pLocalWorkSize
       desc: "[in][optional] Local work size to use when executing kernel."
+    - type: uint32_t
+      name: "numKernelAlternatives"
+      desc: "[in] The number of kernel alternatives provided in pKernelAlternatives."
+    - type: $x_kernel_handle_t*
+      name: "phKernelAlternatives"
+      desc: |
+            [in][optional][range(0, numKernelAlternatives)] List of kernels handles that might be used to update the kernel in this 
+            command after the command-buffer is finalized.
```

Comment:
Rather than adding the list of kernels as a parameter, I think we should have a new descriptor parameter to the entry-point and then specify the kernel list as part of that. It gives more extensibility without API breakages.

To do that we'll need to first define something like:
```
name: $x_exp_command_buffer_kernel_command_desc_t
base: $x_base_desc_t
members:
    - type: uint32_t
      name: numKernelAlternatives
      desc: foo
    - type: $x_kernel_handle_t*
      name: phKernelAlternatives
      desc: "bar"
```

Then the new param would be like `const ur_exp_command_buffer_kernel_command_desc_t* pKernelDesc`